### DISCUSSION TITLE
Next Dev Session (15 JAN 22)

### DISCUSSION THREAD

**Discussion by tpike3** [mesa/mesa#1094] [General]
This discussion will determine the time for the next dev session to optimize for can attend based on global contributors.

**Comment by jackiekazil** [comment]
Proposal: 8am EST 
https://everytimezone.com/convert/us_eastern_time/8am?t=61be7600,30c

**Comment by tpike3** [comment]
With the next planned dev session in less than a week I would like to confirm who can make it. 

**When**: It is planned for 1400 GMT /2200 UTC +8 (Singapore) / 0800 UTC -5 (EST) on 8 Jan

**Where**: It will be via videochat link to be posted. 

**Who**:
Please know everyone who is interested is invited. 

Of the individuals who have been recent contributors or shown interest, please let us know
@jackiekazil
@dmasad 
@rht  
@danielxu05
@EwoutH 
@jonahaag 
@jdfoote 

**What**: We will look at current pull reuqests, new issues and discuss the next Mesa release, with focus on self.empties choices and batchrunner.

**Reply by jackiekazil** [reply]
I will be there.

**Reply by rht** [reply]
I will be there too. Also, I recommend using meet.jit.si over Google Hangout, the former being free software.

**Reply by EwoutH** [reply]
I’m traveling Saturday, so unfortunately I won’t be able to join.

**Reply by tpike3** [reply]
Well I have to teach a class Saturday and Sunday morning. So since we are at 50% can 50% can't. Does anyone want to suggest an alternate time and date that may work?  I think anytime this week or next.

**Reply by jackiekazil** [reply]
The following Saturday?

**Reply by tpike3** [reply]
That works for me!

**Reply by danielxu05** [reply]
I will join in the meeting on the following Saturday.

**Comment by tpike3** [comment]
I am reposting with the update.. based on responses we are now planning on meeting on the 15th of January. 

With the next planned dev session in less than a week I would like to confirm who can make it.

When: It is planned for 1400 GMT /2200 UTC +8 (Singapore) / 0800 UTC -5 (EST) on 15 Jan

Where: [via Jitsi](https://www.google.com/url?q=https://meet.jit.si/SpecificDependencesMaximizeAlso&sa=D&source=calendar&usd=2&usg=AOvVaw0WPWuakM5XiZogcBbr-9m0)

Who:
Please know everyone who is interested is invited.

Of the individuals who have been recent contributors or shown interest, please let us know
@jackiekazil
@dmasad
@rht
@danielxu05
@EwoutH
@jonahaag
@jdfoote

What: We will look at current pull reuqests, new issues and discuss the next Mesa release, with focus on self.empties choices and batchrunner.

**Reply by EwoutH** [reply]
Sorry for the late response, things have been shaken up quite a bit by a recent quarantine (I'm perfectly ok). I can attend tomorrow at 14:00 UTC.

**Reply by rht** [reply]
I forgot to confirm that I'm joining in.

**Reply by tpike3** [reply]
Excellent I am in the room
